binding.gyp

```diff
@@ -6,7 +6,24 @@
   'targets': [
     {
       'target_name': 'iconv',
-      'include_dirs': ['<!(node -e "require(\'nan\')")'],
+      'cflags!': [ '-fno-exceptions' ],
```

Comment:
The original version of node-iconv made sure there were no CPP exceptions. Thus, we should maintain that choice. This means that `cflags!` needs to be changed to `cflags` and `cflags_cc!` needs to be changed to `cflags_cc`.